ecommerce/api.py

```diff
@@ -88,7 +93,18 @@ def create_unfulfilled_order(course_id, user):
         Order: A newly created Order for the CourseRun with the given course_id
     """
     course_run = get_purchasable_course_run(course_id, user)
-    price = course_run.courseprice_set.get(is_valid=True).price
+
+    price_dict = get_course_price_for_learner(user, course_run.course.program)
+    price = price_dict['course_price']
```

Comment:
Yes, the function has only one return statement and the key is defined in the dict there